lsl_autobids/dataverse_dataset_create.py

```diff
@@ -40,70 +61,35 @@ def create_dataverse(project_name):
 
     # open the toml file to get the dataset_id
     toml_path = os.path.join(project_root,project_name,project_name+'_config.toml')
-    with open(toml_path, 'rb') as file:
-        data = tomllib.load(file)
-        pid = data['Dataverse']['pid']
-
-        if pid.lower() in pids_resp1:
-            flag=1
-            print('Dataset already exists.')
-            return pid,flag
-        else:
-            print('Creating the dataset........')
-            resp = api.create_dataset(parent_dataverse_name, ds.json())
-            print(resp.json()['data'])
-            ds_pid = resp.json()['data']['persistentId']
-            dataset_id = resp.json()['data']['id']
-            print(f"Dataset created with pid: {ds_pid}")
-
-
-            # Modify field
-            data['Dataverse']['dataset_title']=ds_title 
-            data['Dataverse']['pid']=ds_pid
-            data['Dataverse']['dataset_id']=dataset_id
-
-            # To use the dump function, you need to open the file in 'write' mode
-            # It did not work if I just specify file location like in load
-            f = open(toml_path,'w')
-            toml.dump(data, f)
-            f.close()
-            
-            # # change the dataset_id in the yaml file
-            # data['pid'] = ds_pid
-            # data['dataset_id'] = dataset_id
-            # data['dataset_title'] = ds_title
-
-            # # write the updated yaml file
-            # with open(project_path + '/dataset_data.yaml','w+') as yaml_file:
-            #     yaml.dump(data,yaml_file)
-            return ds_pid,flag
-            
-
-
-
-
-
-    # Publish Dataset if you want to publish it
-    #resp = api.publish_dataset(ds_pid, release_type="major")
-
-    #return ds_pid
-
-# def ask_for_data_upload():
-#     while True:
-#         response = input(" Do you want to upload any data file? Please type 'yes' or 'no': ").lower()
+   
+    data = read_toml_file(toml_path)
+    pid = data['Dataverse']['pid']
+
+    if pid.lower() in pids_resp1:
+        flag=1
```

Comment:
Could we have const flags which are meaningful? I myself confused myself a lot when trying to remember what flag 0/1/2 meant 🙈